### DISCUSSION TITLE
友友们，起猛了，什么是极致的压缩啊

### DISCUSSION THREAD

**Discussion by fairy-B** [share121/inter-knot#1857] [其他]
# 文件下载网址：https://unforgettable.dk/
#### 进入后点击下图所指的
![image](https://github.com/user-attachments/assets/7ea0b302-eade-4d46-bdb5-430f7d6ddd28)

位置42kb的压缩文件，解压密码是：42
电脑内存不够4.5PB的友友们不要解压啊！

**Comment by lllliufeng** [comment]
电脑：你不要过来啊

**Comment by capacitor1** [comment]
试过了。没办法自动解压，还是得手动，炸不了

**Comment by CloudSwordSage** [comment]
唔，被拦截了，找个时候用虚拟机玩玩，本机有点方
---
![image](https://github.com/user-attachments/assets/bd08cde0-d876-4eae-96de-2e227ff804b5)

**Comment by fairy-B** [comment]
楼上火绒立大功（
